crates/context_aware_config/Cargo.toml

```diff
@@ -11,6 +11,7 @@ actix-http = "3.3.1"
 actix-web = { workspace = true }
 anyhow = { workspace = true }
 base64 = { workspace = true }
+bigdecimal = { version = "0.3.1" , features= ["serde"]}
```

Comment:
Do we need bigdecimal?

Reply:
Yes @Datron we need to calculate higher value